distributed/client.py

```diff
@@ -937,6 +974,11 @@ def __init__(
 
         ReplayTaskClient(self)
 
+    def _set_default_configs(self):
```

Comment:
This _set_default_configs shouldn't be needed now it's managed by _GlobalClientManager